bin/admin/install-solvcon.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [ $# -lt 1 ]
+then
+    echo "Usage : $0 <solvcon env configuration file>"
+    exit
+fi
+solvcon_env_file=$1
+git clone https://github.com/solvcon/solvcon.git
+
+# you never know the env is ready to use SOLVCON or not, so
+# enable the env anyway.
+source $solvcon_env_file
+# basic env
+export SCSRC=$HOME/solvcon
```

Comment:
I don't think it's necessary to set SCSRC here.  In https://github.com/solvcon/solvcon/blob/master/contrib/devenv/start#L2 , SCSRC is already set by the startup script.

Hardcoding SCSRC here prevents installing multiple versions  of SOLVCON on one host.  But in development it's very likely that we'll need multiple versions.